src/common/utils/get-startable-stoppable.ts

```diff
@@ -3,58 +3,85 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
-type Stopper = () => Promise<void> | void;
-type Starter = () => Promise<Stopper> | Stopper;
+export type Stopper = () => Promise<void> | void;
+export type Starter = () => Promise<Stopper> | Stopper;
 
-export const getStartableStoppable = (
```

Comment:
Refactoring this is not implied in name of commit. Add explicit commits in future? :)